deploy/default-azuredeploy.json

```diff
@@ -1,125 +1,133 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "serviceName": {
-            "type": "string",
-            "defaultValue": "[concat('FhirConverter-', utcNow())]",
-            "metadata": {
-                "description": "Name of the FHIR Converter Web App."
-            }
-        },
-        "apiKey": {
-            "type": "secureString",
-            "defaultValue": "[newGuid()]",
-            "metadata": {
-                "description": "API key to access the app."
-            }
-        },
-        "appServicePlanSku": {
-            "type": "string",
-            "allowedValues": [
-                "F1",
-                "B1",
-                "B2",
-                "B3",
-                "S1",
-                "S2",
-                "S3",
-                "P1",
-                "P2",
-                "P3"
-            ],
-        "defaultValue": "B2"
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceName": {
+      "type": "string",
+      "defaultValue": "[concat('FhirConverter-', utcNow())]",
+      "metadata": {
+        "description": "Name of the FHIR Converter Web App."
+      }
+    },
+    "apiKey": {
+      "type": "secureString",
+      "defaultValue": "[newGuid()]",
+      "metadata": {
+        "description": "API key to access the app."
+      }
+    },
+    "appServicePlanSku": {
+      "type": "string",
+      "allowedValues": [
+        "F1",
+        "B1",
+        "B2",
+        "B3",
+        "S1",
+        "S2",
+        "S3",
+        "P1",
+        "P2",
+        "P3"
+      ],
+      "defaultValue": "B2"
+    },
+    "imageTag": {
+      "type": "string",
+      "allowedValues": [
+        "1.0.0",
+        "2.0.0"
+      ],
+      "defaultValue": "2.0.0"

```

Comment:
the versions should be v1.0.0 and v2.0.0

Reply:
Can't do the facepalm emoji here, right when I need it.

Reply:
Never mind. I have tested that both options work perfectly now